documentation/docs/configuration.md

```diff
@@ -242,6 +242,14 @@ to disable. **`Default: ""`**
 ####`get5_warmup_cfg`
 :   Config file executed in warmup periods. **`Default: get5/warmup.cfg`**
 
+####`get5_demo_upload_url`
+:   If defined, it is the URL at which a server resides to accept connections to upload demos. Requires SteamWorks
+extension.
+
+####`get5_demo_upload_token`
+:   If defined, it is the authorization token used to upload to an accepting web server. Requires SteamWorks
```

Comment:
Explain which kind of authorization token, as you've currently fixed it to `Authorization: Bearer %s`

Reply:
It might be better to make the variable "Bearer <token>" so people can use any type of `Authorization` they want.

Reply:
I think the issue with having different authorization is that it would change the type of header that is sent in. For example, a Bearer token has the http key as "Authorization" where an API key has the http key as "Basic". So we could either do a split on the string that's provided by the cvar and attempt to swap the header key with a case/if statement, or have one set authorization type.

Reply:
If you want that, you will have to provide 2 cvars. One can just default to `Authorization` then and the token to blank.